lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -254,6 +258,16 @@ export class BrowserAuthError extends AuthError {
         return new BrowserAuthError(BrowserAuthErrorMessage.popUpWindowError.code, errorMessage);
     }
 
+    /**
+     * Creates an error thrown when the popup window could not be opened because invalid attributes passed in request.
+     * @param errDetail 
+     */
+    static createPopupWindowAttributeError(errDetail?: string): BrowserAuthError {
```

Comment:
I'd recommend against failing in this flow and instead logging an error and using the default size and centered position.  If the developer wants a larger popup than the users screen size then the user would not get a prompt.